lib/phases/reboot.js

```diff
@@ -63,6 +63,7 @@ Reboot.prototype.reboot = function() {
       return phase._restart();
     })
     .then(function(time) {
+      phase.setup();
```

Comment:
We don't really use the time parameter realistically anymore, so we can probably remove the logic. Instead try this:

```js
.then(function() {
  return phase.setup();
})
.then(function() {
  return phase.device.log.mark(phase.START_MARK, Date.now());
})
```